pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -111,6 +111,18 @@ var (
 				owner: "125523088429",
 			},
 		},
+		providerconfigtypes.OperatingSystemRockyLinux: {
+			awstypes.CPUArchitectureX86_64: {
+				description: "Rocky-8-ec2-8.5-20211114.2.x86_64",
```

Comment:
Can we use a wildcard instead of pinning a version similar to what we do for other operating systems?

Reply:
Sure

Reply:
```suggestion
				description: "Rocky-8-ec2-8*.x86_64",
```

yeah, that will automatically use newer version when it's available